@xen-orchestra/lite/src/components/pool/dashboard/ramUsage/PoolRamUsage.vue

```diff
@@ -100,5 +100,11 @@ const isStatFetched = computed(() => {
 
 const isLoading = computed(() => (isFetching.value && !hasError.value) || !isStatFetched.value)
 
-const customValueFormatter = (value: number) => String(formatSize(value))
+const customValueFormatter = (value: number | null) => {
+  if (value === null) {
+    return ''
+  }
+
+  return String(formatSize(value))
```

Comment:
this cast is not necessary
```suggestion
  return formatSize(value)
```